src/api/procedures/__tests__/createSecurityToken.ts

```diff
@@ -129,6 +138,10 @@ describe('createSecurityToken procedure', () => {
       })
     );
     rawFundingRound = polkadotMockUtils.createMockFundingRoundName(fundingRound);
+    rawPosRatio = polkadotMockUtils.createMockPostRatio(numerator, denominator);
+    rawFee = polkadotMockUtils.createMockBalance(fee);
+    numerator = 1;
```

Comment:
You should test with values different than 1 to make sure that the division and multiplication are actually being done